src/seth/libexec/seth/seth-combined-json

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+### seth-combined-json -- alias for solc --combined-json
```

Comment:
Wondering if we need a command to alias a single argument?

Reply:
Oh I see, it's basically `seth --use solc:0.4.11 seth combined-json`, sorry misread the comment.

I think if `seth` starts having subcommands for compiling stuff, it should mention that in the subcommand name.

`seth combined-json` for me doesn't make it clear that it's calling solc.

But I wonder if seth should call solc in the first place. Until now, IIUC, that was dapp's job. seth is for interacting with the network.

Reply:
To be clear, its `--combined-json=abi,ast,storage-layout,srcmap,srcmapruntime...` basically, its "compile solidity with as much artefacts as possible", and also the output that hevm expecys

Reply:
It is true that dapp has handled the `nix` and `solc` stuff in the past, which is why I'd like to get some feedback on how people feel about this. `seth debug` could be a `dapp tx-debug` command or similar, I just don't find it as catchy. It has more of a seth feel to it imo

Reply:
but these lines are definitely getting blurry